.circleci/build-assets.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+
+echo "Begin build for $CIRCLE_ENV${PR_ENV:+ for }$PR_ENV. Pantheon test environment is $TERMINUS_SITE.$TERMINUS_ENV"
+
+# Build assets with Composer
+composer -n build-assets
+
+# Compile Sass or run any other build steps necessary
```

Comment:
In the Drupal examples, we encourage folks to add their other steps to `build-assets` in composer.json. Could we do that here as well, for consistency?  Ideally, we'd like it if the build step was called in the same way regardless of whether you are using Circle or Travis or Jenkins or other for testing -- in other words, just call `composer -n build-assets` directly where needed.